### PR TITLE
LPS-149317 Fix tag closing syntax error

### PR DESCRIPTION
**Pull request by matuzalemsteles**
Basically, these are the fixes that we fixed in PR #1986, but the error of the problem was happening because the SSR of the taglib progress bar was not closing the tag correctly.

You can reproduce this behavior by editing the DXP main page, adding a new Clay Sample widget, and trying to navigate the Stickers tab, without this PR it will not render anything with this fix it will show the stickers examples.

Basically, the stickers examples were being rendered but the incorrect tag closing caused the stickers tab to be rendered inside the progress bar example.

### CODE REVIEW

**Review by julien — APPROVED** [review]
LGTM - thanks for the fix @matuzalemsteles

**Comment by julien** [comment]
@matuzalemsteles the semantic versioning error is not unique to this pull request, @susanchen3 kindly created a [ticket](https://issues.liferay.com/browse/LRQA-73827) to address this. I'm not sure about the `yarnInstall` issue but I know there was a CI outage yesterday. I'll give this another try

**Comment by julien** [comment]
ci:test:relevant

**Comment by susanchen3** [comment]
The `yarnInstall` issue looks like it was a fluke. The other test failures look unrelated. Looks good to forward :) :heavy_check_mark:

**Comment by matuzalemsteles** [comment]
I'll try with `ci:forward` if it still fails I'll send it manually.

**Comment by matuzalemsteles** [comment]
ci:forward

**Comment by matuzalemsteles** [comment]
Manually forwarded https://github.com/brianchandotcom/liferay-portal/pull/114762
